src/tasks/purchase/taskpurchase.cpp

```diff
@@ -56,9 +58,18 @@ TaskPurchase::TaskPurchase(Op op) : Task("TaskPurchase"), m_op(op) {
 TaskPurchase::~TaskPurchase() { MZ_COUNT_DTOR(TaskPurchase); }
 
 void TaskPurchase::run() {
+#ifdef MZ_IOS
+  // APIv2 returns 200, APIv1 returns 201
```

Comment:
This is a relatively new addition to the VPN codebase, but you can also do:
```
request = new NetworkRequest(this);
request->addExpectedStatus(200);
request->addExpectedStatus(201);
```
In which case, the `NetworkRequest` will accept either status code as a successful result. We had a need for that in the Balrog handler, because we could get multiple status codes back from the update API.

Reply:
In each scenario, we have only one expected status - I'd rather fail if we get a 200 for APIv1, as that is unexpected. What do you think about that? Is that too strict?